pkg/securitycontextconstraints/sccmatching/provider.go

```diff
@@ -191,9 +191,20 @@ func (s *simpleProvider) CreateContainerSecurityContext(pod *api.Pod, container
 		sc.SetReadOnlyRootFilesystem(&readOnlyRootFS)
 	}
 
+	isPrivileged := sc.Privileged() != nil && *sc.Privileged()
```

Comment:
do you know why `sc.Privileged()` doesn't take into account capabilities?

Reply:
ah, okay this is just a flag, right?

Reply:
it's only an accessor for the privileged field. Maybe my local var is misnamed.

Reply:
`isHighlyPrivileged` or `isRoot` maybe?

Reply:
`isPrivileged` and `addCapSysAdmin`